src/plots/geo/constants.js

```diff
@@ -109,28 +107,28 @@ exports.projNames = {
     'winkel3': 'winkel3',
 
     // 'interrupt': 'interrupt',
-    'interrupted homolosine': 'interruptedHomolosine',
-    'interrupted sinusoidal': 'interruptedSinusoidal',
-    'interrupted boggs': 'interruptedBoggs',
-    'interrupted sinu mollweide': 'interruptedSinuMollweide',
-    'interrupted mollweide': 'interruptedMollweide',
-    'interrupted mollweide hemispheres': 'interruptedMollweideHemispheres',
-    'interrupted quartic authalic': 'interruptedQuarticAuthalic',
+    // 'interrupted homolosine': 'interruptedHomolosine',
+    // 'interrupted sinusoidal': 'interruptedSinusoidal',
+    // 'interrupted boggs': 'interruptedBoggs',
+    // 'interrupted sinu mollweide': 'interruptedSinuMollweide',
+    // 'interrupted mollweide': 'interruptedMollweide',
+    // 'interrupted mollweide hemispheres': 'interruptedMollweideHemispheres',
+    // 'interrupted quartic authalic': 'interruptedQuarticAuthalic',
 
-    'polyhedral butterfly': 'polyhedralButterfly',
-    'polyhedral collignon': 'polyhedralCollignon',
-    'polyhedral waterman': 'polyhedralWaterman',
+    // 'polyhedral butterfly': 'polyhedralButterfly',
+    // 'polyhedral collignon': 'polyhedralCollignon',
+    // 'polyhedral waterman': 'polyhedralWaterman',
 
     'gringorten quincuncial': 'gringortenQuincuncial',
     'peirce quincuncial': 'peirceQuincuncial',
-*/
 };
 
 // name of the axes
 exports.axesNames = ['lonaxis', 'lataxis'];
 
 // max longitudinal angular span (EXPERIMENTAL)
 exports.lonaxisSpan = {
+    'satellite': 120,
```

Comment:
TODO: modify this according to https://github.com/plotly/plotly.js/issues/4781#issuecomment-619886963